aws/services/lb/resource.ftl

```diff
@@ -222,12 +234,12 @@
                 {
                     "Certificates" : [
                         {
-                            "CertificateArn" : getReference(certificateId, ARN_ATTRIBUTE_TYPE, regionId)
+                            "CertificateArn" : certificateArn
                         }
                     ],
                     "SslPolicy" : sslPolicy
                 },
-                port.Certificate!false)
+                (port.Certificate!false && certificateArn?has_content))
```

Comment:
I don't think we need to exclude including the resource content if we are going to throw a fatal error. Debugging issues would be a bit weird if that was the case

Reply:
Yep, i was 50/50 on that one. I'll remove it.

Reply:
Removed in efe8bb8